frontend/src/metabase/dashboard/actions/sharing.ts

```diff
@@ -29,7 +29,11 @@ export const UPDATE_ENABLE_EMBEDDING =
 export const updateEnableEmbedding = createAction(
   UPDATE_ENABLE_EMBEDDING,
   ({ id }: DashboardIdPayload, enable_embedding: boolean) =>
-    DashboardApi.update({ id, enable_embedding }),
+    DashboardApi.update({
```

Comment:
I'm sure there's a way to do this on the BE so we can make sure the DB type and the timezone-aware string work normally. We could probably even send the `static_embed_published` and `static_embed_unpublished`  snowplow event from the BE but I'll need some guidance on that

Reply:
yeah i think for now this is ok. From what I understand we have a few better utils for handling things like this on the FE but we can talk with some BEs in the future to make this nicer. It works for now